### PR TITLE
Updating repo guidelines for removing product specific glossary

### PR DESCRIPTION
**Pull request by bergerhoffer**
- Updated guidelines to remove product-specific section info
- Updated history to remove broken links to product-specific sections
- Removed product-specific files (so that people don't accidentally update the wrong sections in the future)

**NOTE:** Since I'm deleting files, I created a backup branch called [backup-files-before-reorg](https://github.com/redhat-documentation/supplementary-style-guide/tree/backup-files-before-reorg) that has all of the product term files in case we ever need to find/reference them in the future.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council For review please!

File: GUIDELINES.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ Use the following template for glossary entries:
...
 |`<term>`               |The term to be described.          |
 |`<class>`              |The part of speech of the term.|Possible values: noun, verb, adjective, adverb, pronoun, conjunction, preposition. Use noun for acronyms or abbreviations. Leave the field empty for symbols.
-|`<description>`        |The description of the term.|Use full sentences with periods. If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
+|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```
Comment (porueesq) [inline GUIDELINES.adoc:36]:
```suggestion
|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

**Review by porueesq — DISMISSED** [review]
A suggestion for your consideration. Not sure if the removed sentence is redundant, or needs additional words.

File: GUIDELINES.adoc (hunk 1 above)
Comment (bergerhoffer) [reply GUIDELINES.adoc:36]: @porueesq As far as I can tell this isn't a duplicate sentence. Can you explain which other sentence you think it duplicates?

The one I think you might be looking at is the last one about the Incorrect forms. If so, that one is talking about adding reasons why the values in the "Incorrect forms" field are incorrect. But this one is talking about if the "use it" field is set to no, explain why the term shouldn't be used. For example: https://redhat-documentation.github.io/supplementary-style-guide/#puppetize

Comment (mportman12) [inline GUIDELINES.adoc:36]:
```suggestion
|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should or shouldn't use the term in this form. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

**Review by mportman12 — APPROVED** [review]
Left a wording suggestion (optional), otherwise LGTM

File: GUIDELINES.adoc (hunk 1 above)
Comment (bergerhoffer) [reply GUIDELINES.adoc:36]: Thanks @mportman12 that works, I'll make that change

Comment (porueesq) [reply GUIDELINES.adoc]: @bergerhoffer Thanks for the explanation. Then, this should be clarified in the text as well to clearly distinguish these cases: 
* why a term is used
* why the same term isn't used
* why incorrect forms of the original term are considered as such



```suggestion
|`<description>`  |The description of the term. |Use full sentences with periods. Add information about why we should use the term in this form. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. When the term in this form is not appropriate for usage, explain the reasons. Alternatively, explain why we consider the terms listed in the *Incorrect forms* field as erroneous.
```

File: GUIDELINES.adoc [hunk 2]
```diff
@@ -33,7 +33,7 @@ Use the following template for glossary entries:
...
 |`<term>`               |The term to be described.          |
 |`<class>`              |The part of speech of the term.|Possible values: noun, verb, adjective, adverb, pronoun, conjunction, preposition. Use noun for acronyms or abbreviations. Leave the field empty for symbols.
-|`<description>`        |The description of the term.|Use full sentences with periods. If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
+|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should or shouldn't use the term in this form. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```
Comment (ndeevy) [inline GUIDELINES.adoc:36]: Should these variables use underscores instead of hyphens here:

 (1) <first-meaning>. (2) <second-meaning>.

**Comment by ndeevy** [comment]
One insignificant nitpicky question, otherwise LGTM.

**Review by ndeevy — APPROVED** [review]

**Review by mportman12 — APPROVED** [review]

File: GUIDELINES.adoc (hunk 1 above)
Comment (bergerhoffer) [reply GUIDELINES.adoc]: @porueesq I had updated this section per miriam's suggestion to be simpler. Can you let me know if that update was sufficient, or if you think we need to make these updates still, thanks!

File: GUIDELINES.adoc (hunk 2 above)
Comment (bergerhoffer) [reply GUIDELINES.adoc]: Good catch @ndeevy. I updated all other instances of replaceable values in the file that used hyphens and changed them to underscores. PTAL when you get another chance!

File: GUIDELINES.adoc (hunk 1 above)
Comment (porueesq) [reply GUIDELINES.adoc]: Sounds good. I'll approve the MR.

**Review by porueesq — APPROVED** [review]

**Comment by bergerhoffer** [comment]
Thanks @porueesq!

@mportman12 @ndeevy can you take another look and reapprove if you're still good with the changes?

**Review by ndeevy — APPROVED** [review]

File: GUIDELINES.adoc (hunk 2 above)
Comment (ndeevy) [reply GUIDELINES.adoc]: Done. Looks great!

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council Can I get another review/approval on this please?

**Comment by dfitzmau** [comment]
Hi @mportman12 . Just your approval is pending on this PR. Are you OK to approve or does something needs to be addressed?

**Pull request merged**